cognite/powerops/resync/models/v2/dms/AFRRBids/containers.yaml

```diff
@@ -0,0 +1,316 @@
+- space: '{{model_space}}'
+  externalId: BidDocument
+  name: BidDocument
+  usedFor: node
+  properties:
+    name:
+      type:
```

Comment:
Should we merge this container and the one with the same name in Day ahead model?

Reply:
How were you thinking to merge these? Were you thinking that the BidDocument container would have optional fields for total, partials and bids? Would it be an issue that the PriceArea nodes are different concepts for both? I can see the rationale, I am just not really sure what this would look like in terms of the containers.

Reply:
The shared properties should be in the same container, and then, have an extra container for the specialized properties.

Reply:
Alright, thanks for taking a look! I will make this change and request another review

Reply:
@frnhr - updated this following the guidance from @doctrino. I also updated the DayAheadBid DMS - which we can move to your branch if you agree.